testing/rundeck/APKBUILD

```diff
@@ -0,0 +1,76 @@
+# Contributor: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+# Maintainer: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+pkgname=rundeck
+pkgver=2.7.2
+pkgrel=0
+pkgdesc="Job scheduler and runbook automation. Enable self-service access to existing scripts and tools."
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+url="http://rundeck.org"
+arch="all"
+license="ASL 2.0"
+depends="openjdk8-jre-base ncurses5-libs findutils"
+makedepends="openjdk8 bash"
+install="$pkgname.pre-install"
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/rundeck/${pkgname}/archive/v${pkgver}.tar.gz
+	rundeck.initd
+	0001-Fix-paths.patch
+	"
+builddir="$srcdir"/$pkgname-$pkgver
+
+prepare() {
+	cd "$builddir"
+	patch -p0 -i "$srcdir"/0001-Fix-paths.patch || return 1
+}
+
+build() {
+	export JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk
+	export TERM=dumb
+	cd "$builddir"
+	bash -c "$builddir/gradlew build"
```

Comment:
it also runs unit tests -- these should be postponed until the check() function is called.